website/blog/2025-11-10-release-1.23.md

```diff
@@ -0,0 +1,380 @@
+---
+title: Podman Desktop 1.23 Release
+description: Podman Desktop 1.23 has been released!
+slug: podman-desktop-release-1.23
+authors: [gastoner]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.23.png
+---
+
+import ReactPlayer from 'react-player'
+import ThemedImage from '@theme/ThemedImage';
+
+Podman Desktop 1.23 Release! 🎉
+
+![podman-desktop-hero-1.23](/img/blog/podman-desktop-release-1.23/banner.png)
+
+Podman Desktop 1.23 is now available. [Click here to download it](/downloads)!
+
+This release brings exciting new features and improvements:
+
+- **Support for a managed configuration**: Loads a 'managed by' default settings file from a pre-defined location on the user's OS to support a managed configuration.
+- **Customizable columns and Dashboard sections**: Modify the table columns and the sections in Podman Desktop dashboard.
+- **Created Docker context for Podman machine**: Improved compatibility with Docker contexts.
+- **New features in Search Bar component**: Extended capabilites of Search Bar component with even more features.
+- **Added rootless or rootful indicator to Podman machines**: Adds a visual indication in the resources section of whether a Podman machine is rootful or rootless.
+- **Podman Network support**: Manage your Podman networks directly from Podman Desktop.
+
+---
+
+## Release details
+
+### Support for a managed configuration
+
+Podman Desktop now supports managed configuration. Users can add a “managed-by” configuration file to enforce specific settings. This allows system administrators to apply configurations from a global system file. With this release, administrators can deploy a managed settings file to a predefined system location, enabling centralized control over Podman Desktop configurations across an entire organization.
```

Comment:
As this is more an entreprise (or _organisation_) feature I would put in at the end, I think this only concern very few users, so to keep interest in the release note, might be nice to have this at the end.

On this paragraph, I would emphasis more on the "entreprise", the sentence

> Users can add a “managed-by” configuration file to enforce specific settings.

Are not really representative of the purpose of this feature, we should mention _organisation_ managing fleet of computers can now configure Podman Desktop with default settings, for example providing the organisation container registry by default in Podman Desktop without requiring the user to do any setup.